spec/app/ics-020-fungible-token-transfer/README.md

```diff
@@ -131,13 +170,17 @@ function onChanOpenInit(
   version: string) => (version: string, err: Error) {
   // only unordered channels allowed
   abortTransactionUnless(order === UNORDERED)
-  // assert that version is "ics20-1" or empty
+  // assert that version is "ics20-1" or "ics20-2" or empty
   // if empty, we return the default transfer version to core IBC
   // as the version for this channel
-  abortTransactionUnless(version === "ics20-1" || version === "")
+  abortTransactionUnless(version === "ics20-2" || version === "ics20-1" || version === "")
   // allocate an escrow address
   channelEscrowAddresses[channelIdentifier] = newAddress()
-  return "ics20-1", nil
+  if version === "ics20-1" {
```

Comment:
```suggestion
  if version === "" {
  // default to latest supported version
   return "ics20-2", nil
  }
  // If the version is not empty and is among those supported, we return the version
  return version, nil 
}
```
An informational thing about this block of code: in the future if we add "ics20-3", then we will need to add another if to handle the case where version is "ics20-2". While this way, we would only change line 173,176, and the default value without adding a new if.